app/controllers/admin/questions_controller.rb

```diff
@@ -1,3 +1,5 @@
+require_relative '../../presenters/question_presenter'
```

Comment:
Might be useless, since Rails autoload everything in `app/*`.
Does it work without this line?